packages/components/organisms/f-header/src/components/Navigation.vue

```diff
@@ -409,6 +398,7 @@ export default {
     methods: {
         onNavToggle () {
             this.navIsOpen = !this.navIsOpen;
+            this.$refs.countrySelector.closeCountrySelector();
```

Comment:
Does this throw an error if the country selector is not present?

Reply:
Good question! I will check now

Reply:
I wrapped it into a condition:+1: